src/components/Thanks/OptInSteps.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+	<div class="steps-container">
+		<ul class="steps-list">
+			<li
+				v-for="(step, index) in steps"
+				:key="index"
+				class="step-item"
+				:class="{ 'completed': index < currentStep }"
+				:style="getDelayStyle(index)"
+			>
+				<div class="step">
+					<span v-if="index < currentStep" class="tw-flex">
+						<kv-material-icon class="tw-text-white tw-w-1.5 tw-h-1.5" :icon="mdiCheckBold" />
+					</span>
+				</div>
+				<div class="step-text">
+					{{ step.text }}
+				</div>
+				<div v-if="index > 0 && index < steps.length" class="line"></div>
+			</li>
+		</ul>
+	</div>
+</template>
+
+<script>
+import { mdiCheckBold } from '@mdi/js';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+
+export default {
+	name: 'OptInSteps',
+	components: {
+		KvMaterialIcon
+	},
+	props: {
+		weeksToRepay: {
+			type: String,
+			default: ''
+		}
+	},
+	data() {
+		return {
+			mdiCheckBold,
+			currentStep: 1,
+			steps: [
+				{ text: 'Your contribution is received ' },
+				{ text: 'Their loan is fully funded' },
+				{ text: 'They use the money to improve their life' },
+				{ text: `They start repaying you in ${this.weeksToRepay}` }
+			]
+		};
+	},
+	methods: {
+		getDelayStyle(index) {
+			return {
+				animationDelay: `${index * 1}s`
+			};
+		}
+	},
+	mounted() {
+		setTimeout(() => {
```

Comment:
Why is `setTimeout` needed here? Would `nextTick` work? Using `nextTick` might be a more specific solution, if we just need to wait for the next render.